File: Convert_string_to_camel_case/camelCase.js
function toCamelCase(str){
if (str.length== 0){
	return str
}
let arr = str.replace(/_/g, '-').split("-")
let string = arr.shift()
for (i=0; i<arr.length; i++){
	string += arr[i].charAt(0).toUpperCase()+arr[i].slice(1)
}

return string
}

console.log(toCamelCase("a_pippi_Was-Pippi"))

File: Convert_string_to_camel_case/camelCase.py
def to_camel_case(text):
    arr = []
    string= ""
    if text == string: 
        return text
   
    arr = text.replace("_", "-").split("-")

    if text[0].isupper(): 
        for i in range(len(arr)): 
            string +=arr[i].capitalize()
    else: 
        string = arr.pop(0)
        for i in range(len(arr)): 
            string+= arr[i].capitalize()
    return string


    

print(to_camel_case("a_Pippi_Was-Pippi"))

